### PR TITLE
notebooks for nucliadb videos

### PR DESCRIPTION
**Pull request by Ciniesta**
### Description
A couple of notebooks with simple use cases to go with the video content generated for the new site

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

File: docs/notebooks/Comparing different embeddings with NucliaDB.ipynb [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+    "nucliadb\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3cf2a673",
+   "metadata": {},
+   "source": [
+    "## check everything's up and running"
```
Comment (jotare) [inline docs/notebooks/Comparing different embeddings with NucliaDB.ipynb:54]:
```suggestion
    "## Check everything's up and running"
```

File: docs/notebooks/Comparing different embeddings with NucliaDB.ipynb [hunk 2]
```diff
@@ -0,0 +1,587 @@
...
+     },
+     "execution_count": 374,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "response = requests.get(f\"http://0.0.0.0:8080\")\n",
+    "response"
```
Comment (jotare) [inline docs/notebooks/Comparing different embeddings with NucliaDB.ipynb:77]: Maybe better to do an assert like:
```
assert response.status_code == 200, "Ups, it seems something is not properly installed"
```

File: docs/notebooks/Comparing different embeddings with NucliaDB.ipynb [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+   "id": "5efd9d92",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def print_results(model_name, results):\n",
+    "    print(f\"---{model_name.upper()} RESULTS---\")\n",
+    "    count=0\n",
+    "    for result in results:\n",
+    "        if count>=3:\n",
+    "            break\n",
```
Comment (jotare) [inline docs/notebooks/Comparing different embeddings with NucliaDB.ipynb:270]: NIT: might be more idiomatic like this:
```
for count, result in enumerate(results[:3])
```

File: docs/notebooks/Semantic Search in minutes with NucliaDB.ipynb [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+    "nucliadb\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91f01094",
+   "metadata": {},
+   "source": [
+    "## check everything's up and running"
```
Comment (jotare) [inline docs/notebooks/Semantic Search in minutes with NucliaDB.ipynb:53]:
```suggestion
    "## Check everything's up and running"
```

File: docs/notebooks/Semantic Search in minutes with NucliaDB.ipynb [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "39d3ffdd",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```
Comment (jotare) [inline docs/notebooks/Semantic Search in minutes with NucliaDB.ipynb:398]: Remove empty cells from bottom

**Review by jotare — APPROVED** [review]

File: docs/notebooks/Comparing different embeddings with NucliaDB.ipynb (hunk 1 above)
Comment (Ciniesta) [reply docs/notebooks/Comparing different embeddings with NucliaDB.ipynb]: updated

File: docs/notebooks/Comparing different embeddings with NucliaDB.ipynb (hunk 3 above)
Comment (Ciniesta) [reply docs/notebooks/Comparing different embeddings with NucliaDB.ipynb]: Updated, anyway I'll modify this and remove the helper function as soon as we can pass the max number of results we want as a parameter  in the sdk search

**Pull request merged**